Assets/Scripts/RagdollController.cs

```diff
@@ -53,8 +62,35 @@ public void EnableRagdoll(Vector3 knockbackForce)
         rigidbodyToPush.AddForce(knockbackForce, ForceMode.Impulse);
     }
 
-    public void ReparentCamera(Transform cameraTransform)
+    public void ReparentCamera(Camera camera, Transform input)
+    {
+        //camera.transform.parent = cameraParent;
```

Comment:
And this is a live example of an uncommented line :latin_cross:

Reply:
Is fix 👍